CRM/Utils/System/Standalone.php

```diff
@@ -64,6 +64,24 @@ protected function isUserExtensionAvailable(): bool {
     return TRUE;
   }
 
+  /**
+   * Get Url to view user record.
+   *
+   * @param int $contactID
+   *   Contact ID.
+   *
+   * @return string|null
+   */
+  public function getUserRecordUrl($contactID) {
+    if (CRM_Core_Permission::check('cms:administer users')) {
+      $uid = (int) CRM_Core_BAO_UFMatch::getUFId($contactID);
+      if ($uid) {
+        return "/civicrm/admin/user/#?User1=$uid";
```

Comment:
@artfulrobot shouldn't this be going through `Civi::url()`?

Reply:
@artfulrobot that's a good point, but I wonder if it wouldn't be best practice to use the url function anyway, since that supports hooks & any future customizations that might be made.